src/battle_message.c

```diff
@@ -3878,7 +3879,9 @@ struct TrainerSlide
 
 static const struct TrainerSlide sTrainerSlides[] =
 {
-    {0x291, sText_AarghAlmostHadIt, sText_BoxIsFull, sText_123Poof},
+    // Put any trainer slide-in messages inside this array.
+    // Example:
+    // {TRAINER_WALLY_VR_2, sText_AarghAlmostHadIt, sText_BoxIsFull, sText_123Poof},
```

Comment:
I think it may be better to change the example to explicitly call out the slide-in type, e.g.:
```
/* Example
{
    .msgLastSwitchIn = sText_AarghAlmostHadIt,
    .msgLastLowHp = sText_BoxIsFull,
    .msgFirstDown = sText_123Poof,
},
*/
```